.gitignore

```diff
@@ -1,5 +1,6 @@
 /node_modules
 .idea
+.*.swp
```

Comment:
I agree, that ```.*.swp``` belongs in the personal global ```.gitignore```. But isn't this also true for ```.idea```and ```*.sublime-*```? Just wondering...